packages/expo-image/src/Image.types.ts

```diff
@@ -0,0 +1,33 @@
+export enum ImageCacheType {
+  LOCAL_OR_UNKNOWN = 0,
+  REMOTE = 1,
+  DISK = 2,
+  MEMORY = 3,
+}
```

Comment:
`local` was supposed to cover Glide's `DataSource.Local`:
```
public enum DataSource {
  /**
   * Indicates data was probably retrieved locally from the device, although it may have been
   * obtained through a content provider that may have obtained the data from a remote source.
   */
  LOCAL,
```

I agree we should split it into `LOCAL` (this explicit value) and `UNKNOWN` (unknown value). Or maybe `DataSource.LOCAL` should map to `disk`?

When it comes to enums I was torn and I don't remember what is the final convention agreement — @tsapeta, @bbarthec, do you remember what is supposed to be used? `expo-brightness`, which is The Module After Audit also uses integer enums

https://github.com/expo/expo/blob/5b1f4c2722b38ef3cf693f79ced295bc8de99087/packages/expo-brightness/src/Brightness.ts#L7-L11